Scope.Client/Properties/AssemblyInfo.cs

```diff
@@ -24,11 +31,11 @@
 // Version information for an assembly consists of the following four values:
 //
 //      Major Version
-//      Minor Version 
+//      Minor Version
 //      Build Number
 //      Revision
 //
-// You can specify all the values or you can default the Build and Revision Numbers 
+// You can specify all the values or you can default the Build and Revision Numbers
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
 [assembly: AssemblyVersion("1.0.0.0")]
```

Comment:
Since Scope is not fully released I'd recommend 0.1.0.0
See [Semantic Versioning](https://semver.org) specifically the FAQ.

Reply:
Agreed